tracer/src/Datadog.Trace/Ci/Ipc/IpcClient.cs

```diff
@@ -0,0 +1,17 @@
+// <copyright file="IpcClient.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#nullable enable
+
+using System;
+
+namespace Datadog.Trace.Ci.Ipc;
+
+internal class IpcClient : IpcDualChannel
+{
+    public IpcClient(string name)
+        : base($"{name}.send", $"{name}.recv")
```

Comment:
Thinking ahead, I assume for our usages we need this to use a unique value of `name` per "CI Vis process". So are you going to be generating a random name in the `dotnet test` process, and then passing that via an env var to the child processes?

Reply:
I'm using the TestSession Id, that I'm already passing over env-var as a context propagation. That's in the following PR.

Reply:
Is there _any_ chance we could spawn multiple child processes with the same `TestSession` Id? And if so, how do they handle the cases where we have multiple readers and writers on the same channel? 🤔

Reply:
we could spawn multiple child process, the challenge here in this case is related to the child reader, they are protected by mutex, so no collision problems, but the only thing to take into consideration is that only one of the child will read the message sent by the server. 

Is not the case in CI visibility where the child process only write, and we have a single reader in the server.

The recomendation if we want to support this scenario is actually create different channels, one for each child.

Reply:
I'm ok on blocking with a flag in the headers, but let's do that in a different PR 🙏🏻  😃

Reply:
Sounds good to me 👍